PowerVCF.psm1

```diff
@@ -294,12 +294,10 @@ Function Add-VCFApplicationVirtualNetwork {
 }
 Export-ModuleMember -Function Add-VCFApplicationVirtualNetwork
 
-######### End APIs for managing Application Virtual Networks ##########
-
-
+#EndRegion AVN-Management
 
-######### Start APIs for managing Backup and Restore ##########
 
+#Region BackupRestore-Management
```

Comment:
What about using the Region/EndRegion capability but leaving the headings as follows:

#Region APIs for managing Backup and Restore
.....
#EndRegion APIs for managing Backup and Restore

This achieves the same goal you were looking to add into the code file but maintains the descriptions as per the API guide.

Reply:
I was under the impression that it was not possible to use spaces on region names (from past uses).

That's why I came up with those suggested abbreviations. That said, I just tried and it seems to work just fine with a longer sentence such the one you've suggested. I'll update with a new commit and also fix the slipped validateJsonInput change (came from my other branch where I'm enhancing the function).